### PR TITLE
Add test for SElinux contexts

### PR DESCRIPTION
**Pull request by VladimirSlavik**
All files on the new system should have correct contexts.

@rvykydal, can you please take a look at this?

### CODE REVIEW

File: selinux-contexts.ks.in [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+bootloader --timeout=1
+zerombr
+
+clearpart --all --initlabel
+autopart
+
+keyboard us
+lang en
+timezone America/New_York
```
Comment (poncovka) [inline selinux-contexts.ks.in:16]: These look like defaults. Please, consider to use fragments instead.

File: selinux-contexts.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Vladimir Slavik <vslavik@redhat.com>
+
+TESTTYPE="storage"
```
Comment (poncovka) [inline selinux-contexts.sh:20]: Please, use `"security selinux"`.

Comment (VladimirSlavik) [reply selinux-contexts.sh:20]: Than you, done.

File: selinux-contexts.ks.in (hunk 1 above)
Comment (VladimirSlavik) [reply selinux-contexts.ks.in]: Interesting, I did not see the other tests use them... much. Changed, thank you.

**Pull request merged**